livre/lib/configuration/ApplicationParameters.cpp

```diff
@@ -110,9 +103,6 @@ void ApplicationParameters::initialize_()
     dataFileName = configuration_.getValue( DATAFILE_PARAM, dataFileName );
     transferFunction = configuration_.getValue( TRANSFERFUNCTION_PARAM,
                                                 transferFunction );
-#ifdef LIVRE_USE_ZEQ
-    syncCamera = configuration_.getValue( SYNC_CAMERA_PARAM, syncCamera );
```

Comment:
Because now this is "fixed" by the session filtering in ZeroEQ. This was a WAR for the global session turning cameras between users.